### DISCUSSION TITLE
Create a Github "Project" for 2.0 Roadmap

### DISCUSSION THREAD

**Discussion by lreading** [OWASP/threat-dragon#86] [Ideas] [answered]
We have a pretty clear roadmap for the 2.0 release here: https://owasp.org/www-project-threat-dragon/

While this is great to have, it might be easier to track if we had them as github issues and used the projects feature built into the repository.  This would allow us to better prioritize the issues and keep track of milestones. https://github.com/OWASP/threat-dragon/projects

I'm happy to do the work for that, but I wanted to see if anyone has a better way of doing it or objects to the idea. :)

**Comment by jgadsden** [comment, marked as answer]
Hello @lreading - from my point of view: absolutely yes. Please go ahead and create a project for this in Threat Dragon, and create the issues that you need.

There should be a companion project in TD Desktop ... that is where I tend to be working so I can create that one if you wish?

**Comment by jgadsden** [comment]
I have updated the [roadmap](https://owasp.org/www-project-threat-dragon/#div-roadmap) to show the progress on version 2.0. Feel free to modify and keep it up to date as we go along @andk123 , @lreading and @mike-goodwin .

@lreading I have also added you as a main contributor on the right hand sidebar of https://owasp.org/www-project-threat-dragon/ - I hope that was a correct thing to do but if that is not OK with you then I can remove it. Additionally if you have an email address you want as a link then I can add that, or you can go in and modify as well I guess?

**Comment by jgadsden** [comment]
I have added a version 2.0 project for Threat Dragon desktop application. If all goes to plan then this version of desktop app will import threat dragon and then create the application.

Any extra files for desktop app I would like to add to threat dragon repo so everything is accessible in one place, ins a similar manner to draw.io desktop.
